cmd/flux/cmd/test.go

```diff
@@ -375,7 +412,13 @@ func (t *TestRunner) MarkSkipped(testNames, skips, tags []string, skipUntagged b
 				skip = true
 			}
 		}
-		t.tests[i].skip = skip || skipMap[t.tests[i].Name()] || (skipUntagged && len(t.tests[i].tags) == 0)
+		// skip tests
+		skipTest := false
+		if !skip && len(skips) > 0 {
+			skipTest = containsWithPkgName(skips, t.tests[i])
+		}
+
+		t.tests[i].skip = skip || skipTest || (skipUntagged && len(t.tests[i].tags) == 0)
```

Comment:
This logic is getting harder to follow, now. There is a boolean called `skip` and another called `skipTest`. Can they both be renamed to reflect what they are? Like `skipBecauseTags` and `skipBecauseSkipList`.